disperser/api/proto/encoder/encoder.proto

```diff
@@ -7,7 +7,8 @@ service Encoder {
   rpc EncodeBlob(EncodeBlobRequest) returns (EncodeBlobReply) {}
 }
 
-// BlomCommitments contains the blob's commitment, degree proof, and the actual degree
+// BlobCommitments contains the blob's commitment, degree proof, and the actual degree
+// DEPRECATED: use common.BlobCommitment instead
```

Comment:
Copying this existing scheme in `common` instead of moving because this is currently being used by encoders

Reply:
Once we are fully on v2 we can remove this?

Reply:
Yes!